powerauth-restful-server-spring/src/main/java/io/getlime/security/powerauth/app/rest/api/spring/controller/v2/CustomActivationController.java

```diff
@@ -158,7 +158,7 @@ public void setPowerAuthActivationProvider(CustomActivationProvider activationPr
             return powerAuthApiResponse;
 
         } catch (Exception ex) {
-            logger.warn(ex.getMessage(), ex);
+            logger.warn("Create activation failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.